helm/db-operator/templates/rbac.yaml

```diff
@@ -25,27 +28,34 @@ rules:
   - services
```

Comment:
is database here needed? should already be covered by the api rule (kci.rocks), and think db operator is not create any database. Only verbs in question would be "patch".

Reply:
patch is used to update service see: https://github.com/kloeckner-i/db-operator/blob/master/pkg/controller/database/reconcileProxy.go#L89

Reply:
updated